src/drivers/snapdragon_pwm_out/snapdragon_pwm_out.cpp

```diff
@@ -355,6 +373,10 @@ void task_main(int argc, char *argv[])
 	// Main loop
 	while (!_task_should_exit) {
 
+		if (_mixer) {
+			_mixer->set_airmode(airmode);
```

Comment:
Why does it need to be set at every loop iteration and not only on parameter change?

Reply:
This is currently needed, because a mixer can dynamically be loaded with an `ioctl`. In that case the parameter would not be applied to the new mixer. I agree it's not nice, but it's ok for now.